.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -60,57 +59,55 @@ body:
 
         Please include screenshots (you can drag any image here) where applicable. Redact any sensitive information.
     validations:
-      required: true
+      required: false
   - type: textarea
     attributes:
       label: "System information and logs"
       description: |
-        Provide details about the environment you experienced the issue in. Change msentraid for the provider you are using:
-      value: |
-        ### Environment
-        * broker version: please run `snap info authd-msentraid`
-        * authd version: please run `/usr/libexec/authd version`
-        * gnome shell version: please run `apt policy gnome-shell`
-        * Distribution: (**NAME** in `/etc/os-release`)
-        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
-
-        ### Log files
-        Please redact/remove sensitive information:
-
-        #### Authd entries:
-
-        ```raw
-        journalctl -u authd.service
-        ```
-
-        ####  MS Entra ID broker entries:
-
-        ```raw
-        journalctl -u snap.authd-msentraid.authd-msentraid.service
-        ```
-
-        ### Application settings
-
-        Please redact/remove sensitive information:
-
-        ####  Broker configuration:
-
-        ```raw
-        cat /var/snap/authd-msentraid/current/broker.conf
+        Copy this command and run it in a terminal. It will open a text editor with the system information and logs. Paste the output below. Redact any sensitive information from the logs.
+        ```bash
+        TMPFILE=$(mktemp authd-system-info-XXXXXX.md)
+        if command -v xdg-open > /dev/null && [ -n "${DISPLAY:-}" ]; then EDITOR=xdg-open; elif [ -z "${EDITOR:-}" ]; then EDITOR=editor; fi 
+        cat > "$TMPFILE" <<EOF && $EDITOR "$TMPFILE" && rm -f "$TMPFILE"
```

I wouldn't overwrite `$EDITOR`

```suggestion
        if command -v xdg-open > /dev/null && [ -v DISPLAY ]; then _authd_editor=xdg-open; fi 
        cat > "$TMPFILE" <<EOF && ${_authd_editor:-editor} "$TMPFILE" && rm -f "$TMPFILE"
```

---

>  wouldn't overwrite $EDITOR

Ah, good point. Would be fine in a shell script but since we expect the user to copy-paste this to their terminal, we shouldn't overwrite it. I pushed a fixup commit.

> [ -v DISPLAY ]

That's not POSIX Shell, right? I would prefer to keep it portable.